include/constants/global.h

```diff
@@ -5,6 +5,8 @@
 #include "constants/charcode.h"
 #include "constants/global.fieldmap.h"
 
+#define MAX_MON_MOVES 4
```

Comment:
renaming seems fine but unsure if this should be in constants/global.h or remain in constants/pokemon.h

Reply:
True, I was mostly following how it was structured in pokeemerald.